### PR TITLE
filebeat/input/internal/procnet: fix typos in test error messages

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

Copy/paste errors in test error messages are confusing, so fix them.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: filebeat/input/internal/procnet/procnet_test.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func TestAddrs(t *testing.T) {
 			t.Errorf("unexpected error: %v", err)
 		}
 		if len(addr4) == 0 {
-			t.Errorf("expected addr in addr4 for IPv4 address: %v", addr6)
+			t.Errorf("expected addr in addr4 for IPv4 address: addr6 is %v", addr6)
```
Comment (belimawr) [inline filebeat/input/internal/procnet/procnet_test.go:33]: I'm trying to understand why printing `addr6` here. If the problem is having `addr4` empty, why not just state that?
```suggestion
			t.Error("expected addr in addr4 for IPv4 address, got none)
```

File: filebeat/input/internal/procnet/procnet_test.go [hunk 2]
```diff
@@ -43,10 +43,10 @@ func TestAddrs(t *testing.T) {
 			t.Errorf("unexpected error: %v", err)
 		}
 		if len(addr4) != 0 {
-			t.Errorf("expected addr in addr4 for IPv4 address: %v", addr6)
+			t.Errorf("unexpected addr in addr4 for IPv6 address: %v", addr4)
 		}
 		if len(addr6) == 0 {
-			t.Errorf("unexpected addrs in addr6 for IPv4 address: %v", addr6)
+			t.Errorf("expected addrs in addr6 for IPv6 address: addr4 is %v", addr4)
```
Comment (belimawr) [inline filebeat/input/internal/procnet/procnet_test.go:49]: Same thing here, I don't get why printing `addr4` helps.
```suggestion
			t.Error("expected addrs in addr6 for IPv6 address, got none)
```

**Review by belimawr — CHANGES_REQUESTED** [review]

File: filebeat/input/internal/procnet/procnet_test.go (hunk 2 above)
Comment (efd6) [reply filebeat/input/internal/procnet/procnet_test.go:49]: Same answer.

File: filebeat/input/internal/procnet/procnet_test.go (hunk 1 above)
Comment (efd6) [reply filebeat/input/internal/procnet/procnet_test.go:33]: @belimawr PTAL

**Review by belimawr — APPROVED** [review]

**Pull request merged**